presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -131,6 +131,8 @@ Property Name                                      Description
                                                    to the Hive metastore service.
 
 ``hive.metastore.client.keytab``                   Hive metastore client keytab location.
+``hive.metastore.impersonation.enabled``           Enable metastore end-user impersonation.
+``hive.metastore.impersonation.user``              Default impersonation user when communicating with Hive Metastore
```

Comment:
In what cases we fallback to the default user? It feels like if impersonation is enabled we should never contact the metastore with "default".

Reply:
MetastoreHiveStatisticsProvider#getPartitionsStatistics, it gets the paritition stats.